ouroboros-network-testing/src/Ouroboros/Network/Testing/Data/AbsBearerInfo.hs

```diff
@@ -241,6 +241,7 @@ instance Arbitrary AbsBearerInfo where
       | a <- shrink (abiSDUSize abi)
       ]
 
+
```

Comment:
This change seems unrelated to this commit (introduced in `Added github actions/nix testing`).

Reply:
Yeah, I'll remove it, I added it just to trigger hydra to build and not use cache.